src/accounts/context/userAccount.tsx

```diff
@@ -65,6 +78,11 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
   const [userAccounts, setUserAccounts] = useState<UserAccount[]>(null)
   const [defaultAccountId, setDefaultAccountId] = useState<number>(null)
   const [activeAccountId, setActiveAccountId] = useState<number>(null)
+  const [accountDetails, setAccountDetails] = useState<Account>(null)
+  const [accountDetailsStatus, setAccountDetailsStatus] = useState(
+    RemoteDataState.NotStarted
+  )
+  const {account} = useSelector(selectCurrentIdentity)
```

Comment:
Is there a `selectCurrentAccount` selector?

Reply:
yes indeed. I'll switch it out